python-frontend/src/main/resources/org/sonar/python/types/third_party_protobuf/Xlib.ext.record.protobuf

```diff
@@ -39,17 +39,17 @@ GetVersionXlib.ext.record.GetVersion"Xlib.protocol.rq.ReplyRequestj36j37j3
 RegisterClientsXlib.ext.record.RegisterClients"Xlib.protocol.rq.Requestj36j37j38j39j310j311j
 UnregisterClients!Xlib.ext.record.UnregisterClients"Xlib.protocol.rq.Requestj36j37j38j39j310j311a
 
-GetContextXlib.ext.record.GetContext"Xlib.protocol.rq.ReplyRequestj36j37j38j39j310j311�
-
EnableContextXlib.ext.record.EnableContext"Xlib.protocol.rq.ReplyRequest*�
+GetContextXlib.ext.record.GetContext"Xlib.protocol.rq.ReplyRequestj36j37j38j39j310j311�
```

Comment:
If I build the stubs on my machine, I get instability on the protobuf. We can discuss it offline